packages/rpc-core/src/rpc-methods/getLeaderSchedule.ts

```diff
@@ -0,0 +1,43 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+
+import { Commitment, Slot } from './common';
+
+/**
+ * This return type is a dictionary of validator identities, as base-58 encoded
+ * strings, and their corresponding leader slot indices as values
+ * (indices are relative to the first slot in the requested epoch)
+ * @example
+ * ```json
+ * {
+ *   "4Qkev8aNZcqFNSRhQzwyLMFSsi94jHqE8WNVTJzTP99F": [
+ *     0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20,
+ *     21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38,
+ *     39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56,
+ *     57, 58, 59, 60, 61, 62, 63
+ *   ]
+ * }
+ * ```
+ */
+type GetLeaderScheduleApiResponse =
+    | Readonly<{
+          [key: Base58EncodedAddress]: Slot[];
+      }>[]
```

Comment:
```suggestion
      }>
```

I think you just want a single object here. [Typescript Playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAMhCGATCAnAygYwBYUQVwBsIAlCAZzAHsA7M6AXilKRoJAB4BvAKCj6gDaAawggAXFDLAUAS2oBzALoTqeALYAjVAMUBubgF8AfDu7cA9OagBJKMCxyhdh2SgzXEAB6QMwXFBRyKloIbgwaKQCJOCRUTBx8IlIKCIYoHn4oACJ4DQwsiQEARgAaACYSgGZFEt5+LOQAMwLBABYSgFYSgDZFQzNLKABBakRndzdXDDwUQOpgSWk8XxnQ8NoFz2iEZHRsXEISINSoRgEM+tz8wtKK6trMhohmwvau3sMS9Lq+LPkHFoCADsJQAHCUAJx9Ax9bigSCwHZxfaJI4pEIAMRknn8jGYiFYHAuwlEEiksgUyigqk02j0hiMZnWkRQZW2sT2CUOyWCdCxOLGjAuvyugNuVRqP2yTUBbx60LMQA)

Reply:
Dictionary confused me, ty!